### ISSUE TITLE
[Adblock BUG] Doesnt block podcast ads

### ISSUE THREAD

**Issue by LastLaffs** [CharlieS1103/spicetify-extensions#63]
Adblock does not block podcast ads.

**Comment by RomsYT** [comment]
Same bug.

**Comment by CharlieS1103** [comment]
Not a big, just couldn't figure it out and not many people use podcasts on Spotify

**Comment by RomsYT** [comment]
Ah, I misspoke. I have a problem where audio ads are not blocked when I listen to my music (not podcasts). I tried several ways to fix this problem but none worked.

**Comment by PoorPocketsMcNewHold** [comment]
The thing with those, is that it's directly included into the actual audio feed of the podcast. 
https://community.spotify.com/t5/Content-Questions/Premium-account-and-still-gets-ads-in-podcasts/td-p/5287217
Premium paying users also get those integrated ads. Sadly, there's no easy doable ways to handle this.

**Comment by amd64fox** [comment]
I think what is meant is the regular regional audio ad from Spotify itself, which is activated in some exclusive podcasts for all types of accounts. 

For example, this [podcast](https://open.spotify.com/show/02JzQLXpqTtViFUGQjRkj3?si=NjTLznMmTjeBPPZ8MSuvTA) includes such audio ad.

**Comment by CharlieS1103** [comment]
@amd64fox didnt I try to help you with something similar to this a while ago, did I end up getting it?

**Comment by amd64fox** [comment]
Indeed, there were attempts on your part, but I never received a response from you in Discord.
In SpotX, I am already blocking this audio ad. @jetfir3 helped me; he found a way to block it through a binary file.

**Comment by CharlieS1103** [comment]
@amd64fox mind dming me about said binary file on discord

**Comment by amd64fox** [comment]
For Windows, search for the following in the Spotify.exe file:

```
slots(?=.{3,8}override_url)
```

and replace it with for example
```
slott
```
by doing this, we disrupt the creation of audio ad slots for podcasts.

**Comment by CharlieS1103** [comment]
Oh sick I can implement this in spicetify probably

**Comment by moeengames** [comment]
CharlieS1103 it doesnt block ads anywhere. sometimes it does but most of the times i mute and guess when will the ad finishes

**Comment by rpgdev** [comment]
Getting ads on podcasts too. I didn't use to get them for months but today just got my first, first time playing a podcast on Spotify in weeks too.

**Comment by Fabian42** [comment]
@CharlieS1103 Did that ever happen?

**Comment by CharlieS1103** [comment]
@Fabian42 Nope, likely won't.
